src/goto-programs/goto_program.cpp

```diff
@@ -21,10 +21,16 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <util/invariant.h>
 #include <util/pointer_expr.h>
 #include <util/std_expr.h>
+#include <util/symbol_table.h>
 #include <util/validate.h>
 
 #include <langapi/language_util.h>
 
+std::ostream &goto_programt::output(std::ostream &out) const
+{
+  return output(namespacet(symbol_tablet()), irep_idt(), out);
```

Comment:
As much as this is the same code we had before: is this even safe? The `namespacet` will take a reference to an object that might not exist beyond the lifetime of its constructor.